docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,165 @@
+---
+title: .NET SDK workload sets
+description: "Learn how to use workload sets to control which versions of workloads get installed, updated, or restored."
+author: tdykstra
+ms.author: tdykstra
+ms.service: dotnet
+ms.topic: how-to
+ms.date: 08/02/2024
+
+#customer intent: As a developer using optional workloads, I want to keep my workload versions in sync so that I can avoid conflicts betwween versions. I also want to avoid unexpected workload updates, so that I can avoid disruptions in development.
+---
+<!--https://aka.ms/patterns-feedback-->
+
+# .NET SDK workload sets
+
+The *workload sets* feature provides a version number that represents a group of .NET SDK workloads. The [install](dotnet-workload-install.md), [update](dotnet-workload-update.md), and [restore](dotnet-workload-restore.md) commands use this number in *workload-set update mode* to provide the following benefits:
+
+* You control the cadence of change for installed workload versions. The alternative mode of operation without using workload sets is called *loose manifests update mode*. In this mode, workloads are updated automatically as new versions of individual workloads are released onto any configured NuGet feeds. In workload set update mode, workloads stay at a specific workload set version until you explicitly change that version.
+* You can install and update a combination of workload versions that ship at the same time and are known to work together.
+* You can ensure that everyone on your team is always working on the same workload versions.
+* You don't have to use a rollback file to specify what workload version you want to be on.
+
+<!--Workload sets are published to nuget.org with each release of the .NET SDK, under the package ID `Microsoft.NET.Workloads.<feature band>`-->
+
+Here are some ways you can use workload sets:
+
+* "Pin" the `install` command to a specific workload-set version.
+* Update installed workloads to the latest available workload-set version.
+* Install or update to a specified workload-set version.
```

Comment:
what's the difference between the first and third items in this list?

Reply:
The first is about installing, the third is about updating; I'll remove "Install or" at the start of the third bullet.

Reply:
```suggestion
* Update to a specified workload-set version.
```